### PR TITLE
Add landing message #1 #2

### PR DESCRIPTION
**Pull request by anchristeve**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-  
+  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
```
Comment (pristheo) [inline index.html]: Could you remove `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (anchristeve) [reply index.html]: Sure, can do!

**Review by pristheo — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by pristheo — APPROVED** [review]

**Review by pristheo — APPROVED** [review]
Well done Harry!

**Pull request merged**